packages/sdk/src/RequestController.ts

```diff
@@ -39,6 +45,21 @@ export class RequestController {
 		})
 	}
 
+	public getResultHeaders(): ResultHeaders {
+		return this.resultHeaders
+	}
+	private setResultHeaders(headers: Record<string, any>) {
+		const rateLimit = headers['x-rate-limit-limit'] ?? false
+		const rateLimitRemaining = headers['x-rate-limit-remaining'] ?? false
+
+		if (rateLimit) {
+			this.resultHeaders.rateLimit = parseInt(rateLimit)
+		}
+
+		if (rateLimitRemaining) {
+			this.resultHeaders.rateLimitRemaining = parseInt(rateLimitRemaining)
+		}
```

Comment:
maybe we can reduce this to something like that:
```ts
this.resultHeaders = {
  reateLimit: parseInt(headers['x-rate-limit-limit']) ?? 0,
  rateLimitRemaining: parseInt(headers['x-rate-limit-remaining']) ?? 0
}
```

Reply:
We could, but if something goes wrong our limits will be replaced with 0. Otherwise we always have last valid results with api limits.